server/pulp/server/managers/content/orphan.py

```diff
@@ -261,14 +272,21 @@ def delete_orphan_content_units_by_type(type_id, content_unit_ids=None):
         """
         # get the model matching the type
         content_model = plugin_api.get_unit_model_by_id(type_id)
+        try:
+            unit_key_fields = units_controller.get_unit_key_fields_for_type(type_id)
+        except ValueError:
+            raise MissingResource(content_type_id=type_id)
+
+        fields = ('id', '_storage_path') + unit_key_fields
+        count = 0
```

Comment:
Is this needed? Looks like ``count`` doesn't get used until below where it gets again set to ``0``.